presto-main/src/main/java/com/facebook/presto/operator/DriverContext.java

```diff
@@ -415,6 +423,11 @@ public boolean isPartitioned()
         return partitioned;
     }
 
+    public ScheduledExecutorService getExecutor()
```

Comment:
I think we should name this `getNotificationExecutor` and add a comment that this should only use used for short lived tasks.